action.yml

```diff
@@ -11,6 +11,10 @@ inputs:
     description: Where to look for the Python package to inspect.
     required: false
     default: .
+  skip-wheel:
+    description: Only build the source distribution.
+    required: false
+    default: 'false'
```

Comment:
I assume adding a `skip-sdist:` option should be something that should be taken up in a different PR?